dali/base/dautils.cpp

```diff
@@ -3233,281 +3233,6 @@ void safeChangeModeWrite(IRemoteConnection *conn,const char *name,bool &reload,
     }
 }
 
-
-class CLocalOrDistributedFile: implements ILocalOrDistributedFile, public CInterface
```

Comment:
NB: impl. moved into ws_dfsclient, because it [conditionally] calls wsdfs::lookup, but didn't want dalibase to be dependent on ws_dfsclient lib.